module/VuFind/src/VuFind/Search/EDS/Options.php

```diff
@@ -480,7 +480,10 @@ protected function setOptionsFromConfig()
 
         // Set up highlighting preference
         if (isset($this->searchSettings->General->highlighting)) {
-            $this->highlight = $this->searchSettings->General->highlighting;
+            // For legacy config compatibility, support the "n" value to disable highlighting:
+            $this->highlight = (strtolower($this->searchSettings->General->highlighting) === 'n')
+                ? false
+                : (bool)$this->searchSettings->General->highlighting;
```

Comment:
Wouldn't (bool)"false" turn to true ?

Reply:
Good point -- the .ini parser will correctly handle an unquoted false, but it doesn't hurt to also account for the possibility of a quoted false. I've adjusted accordingly.

Reply:
(Note that there currently seems to be a bit of a delay in GitHub, so pushed changes aren't showing up right away in PRs... hopefully the fix I just made will be visible in a few minutes).

Reply:
Looks good, thanks !